examples/readEvents/README.md

```diff
@@ -0,0 +1,9 @@
+This example reads events from the logs for a smart contract.
+
+Steps:
+1. Rename the ".env.template" file located at the root of the "examples" folder to ".env"
+1. Uncomment, and assign your Ethereum mainnet endpoint to the `RPC_PROVIDER_URL` variable in the ".env" file
```

Comment:
Let's change "Ethereum" to "Starknet".
Also, could you change this example to use a testnet smart contract? It'll be smoother for the user to use only one RPC provider for all their example interactions without the need to change it.